src/controllers/routers/comments.js

```diff
@@ -20,17 +20,17 @@ const {
  * @param {Request} req
  * @param {Response} res
  */
-const getCommentsbyCaseNumber = async (req, res, next) => {
+const getCommentsbyId = async (req, res, next) => {
   const { id } = req.params;
   try {
-    const cases = await Case.query().where('caseNumber', id);
+    const cases = await Case.query().where('id', id);
     if (cases.length === 0) {
       return next(new ResourceNotFound(`Case ${id} does not exist`));
     }
-    const commentByCaseNumber = await Comment.query()
-      .select('message', 'author')
-      .where('caseNumber', id);
-    return res.status(200).json({ comments: commentByCaseNumber });
+    const commentById = await Comment.query()
```

Comment:
please test. 
the endpoint does not work.

And when I debugged so I could test, I found these other issues:
- I'm only getting one message even when I've created multiple messages for a case.
- I'm not getting the username

Reply:
my bad, i tot it was only for create. forgot about get